lib/fastlane/plugin/appcenter/helper/appcenter_helper.rb

```diff
@@ -546,15 +546,23 @@ def self.update_release(api_token, owner_name, app_name, release_id, release_not
       end
 
       # updates release metadata
-      def self.update_release_metadata(api_token, owner_name, app_name, release_id, dsa_signature)
-        return if dsa_signature.to_s == ''
+      def self.update_release_metadata(api_token, owner_name, app_name, release_id, dsa_signature = '', ed_signature = '')
+        return if dsa_signature.to_s == '' && ed_signature.to_s == ''
 
         url = "v0.1/apps/#{owner_name}/#{app_name}/releases/#{release_id}"
-        body = {
-          metadata: {
-            dsa_signature: dsa_signature
-          }
-        }
+        if dsa_signature.to_s != ''
```

Comment:
In my experience, those are not used together therefore this check is either/or.

Reply:
Agreed. You can enforce that with `conflicting_options` and `conflict_block` in the Options list. Or alternatively allow passing both like #222 was doing.

Reply:
@jp-andre Updated to allow both values.